BrightIntosh/AppDelegate.swift

```diff
@@ -236,6 +238,8 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
         })*/
         
         HotKeyUtils.registerHotKey(modifierFlags: UInt32(0 | optionKey | cmdKey) , keyCode: UInt32(kVK_ANSI_B), callback: self.toggleBrightIntosh)
+        HotKeyUtils.registerHotKey(modifierFlags: UInt32(1 | optionKey | cmdKey) , keyCode: UInt32(kVK_ANSI_Minus), callback: self.decreaseGamma)
+        HotKeyUtils.registerHotKey(modifierFlags: UInt32(2 | optionKey | cmdKey) , keyCode: UInt32(kVK_ANSI_Equal), callback: self.increaseGamma)
```

Comment:
I think the 1 and 2 should be 0 too. Actually i believe the zero isn't necessary, so those numbers could be removed entirely.

Reply:
Also i noticed that on my keyboard (with a german layout) those keys aren't the expected ones. I have to press the buttons that would be + and - on an ansi keyboard, but on my keyboard it's `` ` `` and `ß`.

Reply:
I just said 1 and 2 cause 0 was there before and I assumed that was the ID on the broken code, but yeah, why _is_ the ID on a supposed modifier keys there, I'll test with no numbers
and that's a freakin apple issue if the B key works intended, I assume Apple would code character keys like - and = same as keys like B and T, but your experience says it won't, so time for me to debug apple's incompetence I guess 🤣

Reply:
Well i think the keys are identified by their "position" on the keyboard which is described by the corresponding symbol on the ansi layout. (By chance the B key is the same on my layout). So we have to somehow convert symbols to "positions".